pkg/cli/cli.go

```diff
@@ -112,6 +112,7 @@ const (
 	numInstancesFlag          = "num_instances"
 	autoConnectFlag           = "auto_connect"
 	credentialsSourceFlag     = "credentials_source"
+	projectIDFlag             = "project_id"
```

Comment:
Let's also name this `userProjectIDFlag = "user_project_id"` to make it more specific.